easydock/database.py

```diff
@@ -284,7 +283,7 @@ def select_mols_to_dock(db_conn, table_name='mols', add_sql=None):
             yield mol
 
 
-def add_protonation(db_fname, program='chemaxon', tautomerize=True, table_name='mols', add_sql=''):
+def add_protonation(db_fname, ncpu, program='chemaxon', tautomerize=True, table_name='mols', add_sql=''):
```

Comment:
Move `ncpu` to the end with default value 1.
This is a general recommendation to avoid breaking compatibility. New arguments should come with default values and their positions should be at the end. This is not always possible, but if possible it make sense to follow this.